projects/canopy/src/lib/tabs/tabs.component.html

```diff
@@ -1,30 +1,26 @@
 <div class="lg-tabs__list" role="tablist" [attr.aria-label]="label">
-  <lg-heading
```

Comment:
Moving the heading inside the button will cause another accessibility issue because buttons cannot contain heading. The heading should be wrapping the button as it was before 🤔 

Maybe the solution is to use `aria-owns` on the `tablist`. Something like:

```
<div class="lg-tabs__list" role="tablist" [attr.aria-label]="label" aria-owns="tab1-id tab2-id etc">
```

where `tab1-id tab2-id etc` is the button id for all the tabs.

Reply:
Ah yes, and headings cannot contain buttons either: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
Thinking about it, does it even make sense for tabs to have heading markup? It feels like the tabs should just be buttons.

Reply:
Yes, I think we used to have the heading/button structure like that in our project and Grant from DIG once told me there was no real need for the tab to be a heading.

Reply:
Yeah I agree, would have to be a breaking change though. Which is fine, would jus tneed t get that work scheduled in to update it.

Reply:
This sounds good, thanks for flagging @elenagarrone. In terms of the breaking change, we could merge it into the `next` branch and then release alongside the other [upcoming breaking changes](https://github.com/Legal-and-General/canopy/issues) now if needed as Canopy v7, and then leave the Angular upgrade etc for the next major version.

Reply:
@owensgit I thought about it and I think the problem is that it might slow down quite a lot the merging of this fix. We will have to rebase `next` and then merge it and the teams will have to apply all the breaking changes at once.

I guess it depends if this is urgent and if the team (@markblandford) has time to make all the other changes required by the `next` version.